projects/canopy/src/lib/header/account-menu/account-menu-item-label/account-menu-item-label.component.ts

```diff
@@ -0,0 +1,13 @@
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'lg-account-menu-item-label',
+  template: '<ng-content></ng-content>',
```

Comment:
I like that you put the template here instead of creating an unnecessary html file. Maybe we should start doing that more often.